corehq/apps/smsforms/migrations/0003_auto__chg_field_sqlxformssession_couch_id.py

```diff
@@ -0,0 +1,43 @@
+# encoding: utf-8
+import datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+class Migration(SchemaMigration):
+
+    def forwards(self, orm):
+        # Changing field 'SQLXFormsSession.couch_id'
+        db.alter_column(u'smsforms_sqlxformssession', 'couch_id', self.gf('django.db.models.fields.CharField')(max_length=50))
```

Comment:
That should be fine, 400 000 should be pretty quick especially since its
just a scan and not a rewrite of the table.
On 7 Mar 2015 17:33, "Cory Zue" <notifications@github.com> wrote:

> In
> corehq/apps/smsforms/migrations/0003_auto__chg_field_sqlxformssession_couch_id.py
> <https://github.com/dimagi/commcare-hq/pull/5787#discussion_r25996804>:
>
> > @@ -0,0 +1,43 @@
> > +# encoding: utf-8
> > +import datetime
> > +from south.db import db
> > +from south.v2 import SchemaMigration
> > +from django.db import models
> > +
> > +class Migration(SchemaMigration):
> > +
> > +    def forwards(self, orm):
> > +        # Changing field 'SQLXFormsSession.couch_id'
> > +        db.alter_column(u'smsforms_sqlxformssession', 'couch_id', self.gf('django.db.models.fields.CharField')(max_length=50))
>
> thanks for checking into that. i'm inclined to just run this at a low
> traffic time. is about 400,000 records fyi
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dimagi/commcare-hq/pull/5787/files#r25996804>.
>